index.html

```diff
@@ -136,10 +142,18 @@ <h2 class="contributors-head">Contributors</h2>
     </div>
 
     <footer>
-      <p>
-        Page powered by <a href="https://github.com/publiclab/community-toolbox"><i class="fa fa-github"></i></a>
-        <a href="https://github.com/publiclab/community-toolbox">Commmunity Toolbox</a> 
-        - Open Source
+      <p class="links">
+          Page powered by <a href="https://github.com/publiclab/community-toolbox"><i class="fa fa-github"></i></a>
+          <a href="https://github.com/publiclab/community-toolbox">Commmunity Toolbox</a> 
+          - Open Source
+          &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
+  
+          <a href="https://github.com/publiclab"><i class="fa fa-github"></i></a>
+          <a href="https://github.com/publiclab">Public Lab</a> 
+          &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
```

Comment:
Hmm, is it possible to do this via inline CSS instead of all these? if not, no big deal!